mmdet/core/mask/mask_target.py

```diff
@@ -18,6 +18,26 @@ def mask_target(pos_proposals_list, pos_assigned_gt_inds_list, gt_masks_list,
 
     Returns:
         list[Tensor]: Mask target of each image.
+
+    Example:
+        >>> from mmdet.core.mask.mask_target import *  # NOQA
+        >>> import mmdet
+        >>> import mmcv
+        >>> from mmdet.core.mask import BitmapMasks
```

Comment:
We may follow the order of isort to put `import ...` before `from ... import ...`.